test.env.example

```diff
@@ -0,0 +1,11 @@
+SNOWFLAKE_TEST_ACCOUNT=<account>
```

Comment:
```suggestion
SNOWFLAKE_TEST_ACCOUNT=<your Snowflake account's unique identifier>
```

Reply:
The suggested note is confusing to me here. I'm not sure the message you're trying to get across

Reply:
took a stab at note, please let me know if its any more clear we can keep working on the most straight forward take for it.